lib/ship_quiet_logistics/commands/process_shipments.rb

```diff
@@ -0,0 +1,57 @@
+module ShipQuietLogistics
+  module Commands
+    class ProcessShipments
+      def self.call(blackboard:, queue:)
+        new(blackboard, queue).()
+      end
+
+      def initialize(blackboard, queue)
+        @blackboard = blackboard
+        @queue = queue
+
+        @config = ShipQuietLogistics.configuration
+      end
+
+      def call
+        messages_count.times do
```

Comment:
Is this iterator necessary if `next_document` returns falsy when the queue is empty? Is there danger to a `while` loop?